caper/caper_backend.py

```diff
@@ -358,14 +359,17 @@ class CaperBackendLocal(dict):
         }
     }
 
-    def __init__(self, out_dir, concurrent_job_limit=None):
+    def __init__(self, out_dir, concurrent_job_limit=None,
+                 soft_glob_output=False):
         super(CaperBackendLocal, self).__init__(
             CaperBackendLocal.TEMPLATE)
         config = self['backend']['providers'][BACKEND_LOCAL]['config']
         config['root'] = out_dir
 
         if concurrent_job_limit is not None:
             config['concurrent-job-limit'] = concurrent_job_limit
+        if soft_glob_output:
```

Comment:
I agree. Thanks for this. Currently there is no such base class. I will make one since there are other repeating parameters like `concurrent-job-limit`.

Reply:
great!